api/component-library/controllers/component-library.js

```
@@ -0,0 +1,34 @@
+const { ERROR_TYPE, UNIT_TYPE, AUTH_TYPE } = require('../../../config/constants');
+const { throwErrors, getPublicSuperAdmin, isTenantAdmin } = require('../../../config/toolkits');
+
+const { isTruthy,
+        findAllMaterial,
+        handlePublicScope
+} = require('../../../config/material');
+
+module.exports = {
+  async delete(ctx) {
+    const { id } = ctx.params;
+    const entity = await strapi.services['component-library'].delete({ id });
```

ESLint

[error] 'entity' is assigned a value but never used.

(no-unused-vars)